src/components/tables/EntityTable.tsx

```diff
@@ -0,0 +1,355 @@
+                                                    new Date(dateUpdated),
+                                                    { addSuffix: true }
+                                                )}
+                                            </TableCell>
+                                            <TableCell>
+                                                <Box sx={{ display: 'flex' }}>
+                                                    <Button
+                                                        variant="contained"
+                                                        size="small"
+                                                        disableElevation
+                                                        sx={{ mr: 1 }}
+                                                    >
+                                                        Edit
+                                                    </Button>
+
+                                                    {deploymentStatus ===
+                                                    'ACTIVE' ? (
+                                                        <Button
+                                                            variant="contained"
+                                                            size="small"
+                                                            color="error"
+                                                            disableElevation
+                                                            onClick={handlers.deploymentStatusUpdate(
+                                                                catalogNamespace,
+                                                                'INACTIVE'
+                                                            )}
+                                                        >
+                                                            Stop
+                                                        </Button>
+                                                    ) : (
+                                                        <Button
+                                                            variant="contained"
+                                                            size="small"
+                                                            color="success"
+                                                            disableElevation
+                                                            onClick={handlers.deploymentStatusUpdate(
+                                                                catalogNamespace,
+                                                                'ACTIVE'
+                                                            )}
+                                                        >
+                                                            Run
+                                                        </Button>
+                                                    )}
```

Comment:
We could probably combine these buttons and just update the wording and color based on the status.